src/emulator.ts

```diff
@@ -142,6 +152,29 @@ export class EmacsEmulator implements IEmacsController, vscode.Disposable {
   ) {
     this._textEditor = textEditor;
 
+    // Initialize core components
```

Comment:
Splitting the command initializers into multiple sub-methods doesn't make sense.
Putting all lines of the initializers in one place makes it easier to grasp all the initialized commands in order.
Especially splitting `initializeCommandRegistry()` makes things more complex such as introducing type enforcements `!` that are not desirable.